templates/csharp/default-bot/infra/azure.bicep

```diff
@@ -45,24 +44,40 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
         }
         {
           name: 'BOT_ID'
-          value: botAadAppClientId
+          value: identity.properties.clientId
         }
         {
           name: 'BOT_PASSWORD'
```

Comment:
Do we need to keep this `BOT_PASSWORD` key?

Reply:
If I remove this the remote will fail, I'm not sure if it's related to botframeworks's implementation. Let me investigate this.